pyspark-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/pyspark/PySparkInterpreter.scala

```diff
@@ -63,6 +65,7 @@ class PySparkInterpreter(
     )
 
   private lazy val pySparkService = new PySparkService(
+    if (System.getenv(PythonExecEnv) == null) "python" else System.getenv(PythonExecEnv),
```

Comment:
FYI, you could do something like `Option(System.getenv(PythonExecEnv)).getOrElse("python")` instead.